src/components/BottomBar/Controls.js

```diff
@@ -131,40 +133,108 @@ export const Controls = controlsInjector(observer(({ store, history, annotation
 
     const look = (disabled || submitDisabled) ? 'disabled' : 'primary';
 
-    if ((userGenerate && !sentUserGenerate) || (store.explore && !userGenerate && store.hasInterface('submit'))) {
-      const title = submitDisabled
-        ? 'Empty annotations denied in this project'
-        : 'Save results: [ Ctrl+Enter ]';
-      // span is to display tooltip for disabled button
+    const SubmitOption = ({ isUpdate }) => {
+      return (
+        <Button
+          name="list-button"
+          look="secondary"
```

Comment:
onClick for this coming in another ticket?

Reply:
yes working on that now